etc/authordb.yaml

```
@@ -1620,6 +1620,19 @@ affiliations:
     email: null
     institute: Universite de Li\`ege
     ror_id: null
+  LINCC:
+    address:
+      city: Tucson
+      country_code: USA
+      example_expanded: LSST Interdisciplinary Network for Collaboration and 
+        Computing, Tucson, USA
+      postcode: '85721'
+      state: AZ
+      street: 933 North Cherry Avenue
+    department: null
+    email: null
+    institute: LINCC
```

```suggestion
    institute: LSST Interdisciplinary Network for Collaboration and  Computing
```
I think this should be the same string you use in the example (although there is no problem adding "(LINCC)" at the end).

---

Done.